### PR TITLE
DP-1.11: Adding test support for Nokia

### PR DESCRIPTION
**Pull request by trathod1**
Following changes are made in this PR -
    1. Updating deviations in metadata file
    2. Updating test script to support Nokia 

"This code is a Contribution to the OpenConfig Feature Profiles project ("Work") made under the Google Software Grant and Corporate Contributor License Agreement ("CLA") and governed by the Apache License 2.0. No other rights or licenses in or to any of Nokia's intellectual property are granted for any other purpose. This code is provided on an "as is" basis without any warranties of any kind."

### CODE REVIEW

File: feature/qos/ate_tests/bursty_traffic_test/bursty_traffic_test.go [hunk 1]
```diff
@@ -488,6 +488,14 @@ func ConfigureDUTIntf(t *testing.T, dut *ondatra.DUTDevice) {
 		a := s.GetOrCreateAddress(intf.ipAddr)
 		a.PrefixLength = ygot.Uint8(intf.prefixLen)
 		gnmi.Replace(t, dut, gnmi.OC().Interface(intf.intfName).Config(), i)
+		if deviations.ExplicitInterfaceInDefaultVRF(dut) {
```
Comment (cfernanz) [inline feature/qos/ate_tests/bursty_traffic_test/bursty_traffic_test.go:491]: ate testcase was deprecated and removed as per https://github.com/openconfig/featureprofiles/pull/2114 .

Hence, we no longer need this change for ATE

File: feature/qos/ate_tests/bursty_traffic_test/metadata.textproto [hunk 2]
```diff
@@ -27,7 +27,9 @@ platform_exceptions: {
     vendor: NOKIA
   }
   deviations: {
-    explicit_interface_ref_definition: true
```
Comment (cfernanz) [inline feature/qos/ate_tests/bursty_traffic_test/metadata.textproto:30]: same here

ate testcase was deprecated and removed as per https://github.com/openconfig/featureprofiles/pull/2114 .

Hence, we no longer need this change for ATE

File: feature/qos/ate_tests/bursty_traffic_test/bursty_traffic_test.go (hunk 1 above)
Comment (trathod1) [reply feature/qos/ate_tests/bursty_traffic_test/bursty_traffic_test.go]: Fixed branch conflict with removal of ATE changes.

File: feature/qos/ate_tests/bursty_traffic_test/metadata.textproto (hunk 2 above)
Comment (trathod1) [reply feature/qos/ate_tests/bursty_traffic_test/metadata.textproto]: Fixed branch conflict with removal of ATE changes.

**Review by alshabib — APPROVED** [review]

**Pull request merged**